src/pages/app-our-work-single/app-our-work-single.scss

```diff
@@ -0,0 +1,424 @@
+          width: 99%;
+          font-size: 48px;
+        }
+      }
+
+      p {
+        color: #ffffff;
+        font-family: 'Muli';
+        font-size: 18px;
+        font-weight: 400;
+        line-height: 36px;
+        width: 311px;
+        text-align: center;
+
+        @include media-breakpoint-down(sm) {
+          margin-bottom: 23px;
+        }
+      }
+    }
+  }
+
+  section.project-content {
+    margin-top: 80px;
+
+    .images-col {
+      position: relative;
+
+      img {
+        width: 250px;
+        height: 500px;
+
+        @include media-breakpoint-down(md) {
+          width: 130px;
+          height: 260px;
+        }
+
+        @include media-breakpoint-down(sm) {
+          width: 150px;
+          height: 300px;
+          margin-left: 50px;
+        }
+
+        &:first-child {
+          position: absolute;
+          right: 175px;
+
+          @include media-breakpoint-down(md) {
+            position: initial;
+            right: initial;
+          }
+
+          @include media-breakpoint-down(sm) {
+            position: initial;
+            right: initial;
+          }
+        }
+
+        &:last-child {
+          position: absolute;
+          top: 200px;
+          right: 50px;
+
+          @include media-breakpoint-down(md) {
+            top: 100px;
+            left: 100px;
+            right: initial;
+          }
+
+          @include media-breakpoint-down(sm) {
+            top: 80px;
+            left: 80px;
+            right: initial;
+          }
+        }
+      }
+    }
+    h2 {
+      color: #333333;
+      font-family: 'Muli';
+      font-size: 42px;
+      font-weight: 600;
+      letter-spacing: 2.1px;
+      line-height: 47px;
+      text-align: left;
+
+      @include media-breakpoint-down(md) {
+        font-size: 28px;
+      }
+
+      @include media-breakpoint-down(sm) {
+        text-align: left;
+        padding: 0 28px;
+      }
+    }
+
+    p {
+      margin-top: 30px;
+      color: #292a2d;
+      font-family: 'Muli';
+      font-size: 18px;
+      font-weight: 400;
+      line-height: 36px;
+      text-align: left;
+      margin-bottom: 30px;
+      width: 80%;
+
+      @include media-breakpoint-down(md) {
+        font-size: 16px;
+        width: initial;
+      }
+    }
+
+    .project-content-icons {
+      @include media-breakpoint-down(sm) {
+        display: flex;
+        justify-content: center;
+        margin-top: 60px;
+      }
+
+      img {
+        margin: 10px 40px 10px 0;
+
+        @include media-breakpoint-down(sm) {
+          margin: 10px 20px 10px 0;
+        }
+      }
+    }
+
+    .store-buttons {
+      margin-top: 16px;
+
+      @include media-breakpoint-down(sm) {
+        display: flex;
+        justify-content: center;
+        margin-top: 140px;
+      }
+
+      a {
+        margin: 10px 20px 10px 0;
+      }
+    }
+  }
+
+  section.key-features {
+    margin-top: 80px;
+
+    .key-features-text {
+      background-color: #292a2d;
```

Comment:
Let's use the global variable name for this, I believe we have it. If not, let's create it.